_posts/2017-06-09-the-lead-developer-conference.md

```diff
@@ -260,6 +260,44 @@ Katherine brought an interesting and new perspective of (mis)communication betwe
 
 ### Anjuan Simmons - Leadership lessons from the agile manifesto <a name="#Anjuan_Simmons">&nbsp;</a>
 
+Anjuan starts with recounting the greatest influence on his way of being a leader: The Hero with a Thousand Faces, by Joseph Campbell. This book is about the path of the hero through life. Based on the book, Christopher Vogler defines a model for the hero's path. Anjuan summarizes it as follows:
+
+1. The Call to Adventure
+2. Extreme trials
+3. Transformation
+4. Road back
+
+**The call to adventure** is often met with refusal —fear or responsibility in the ordinary life. But eventually the adventurer meets a mentor. She gives the hero both wisdom and gifts to help the hero on her journey. The hero is emboldend to overcome **extreme trials** and level up to approach the final ordeal. Solving this ordeal **transforms** the hero, often portrayed by being given a reward. Next, **the hero returns home**, atoning for the conflicts that were left behind in the ordinary world. Finally she uses her experience and the reward to improve the ordinary world.
```

Comment:
What is the dash in `-fear or responsibility...`?

typo: _emboldened_

I'd rename _Solving this ordeal_ to _Vanquishing_ or _Overcoming_.